pkg/skyenv/skyenv.go

```diff
@@ -78,7 +78,7 @@ const (
 
 	AppSrvAddr                = "localhost:5505" // AppSrvAddr ...
 	ServiceDiscUpdateInterval = time.Minute      // ServiceDiscUpdateInterval ...
-	AppBinPath                = "./apps"         // AppBinPath ...
+	AppBinPath                = "./build/apps"   // AppBinPath ...
```

Comment:
I think we don't need change these, and we just want `make build` command output move from root to  build folder.

Reply:
the apps folder is also to be moved under `./build` as per synth